playbooks/tests/tasks/rally.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Running Rally against test cloud
+  hosts: controller[0]
+  tasks:
+    - name: copy Rally test profile template into place
+      copy: src=rally/bbc-cloud-validate.yml.tmpl dest=./bbc-cloud-validate.yml.tmpl mode=0444
+    - name: copy Rally run script into place
+      copy: src=rally/run.sh dest=./run_rally.sh mode=0555
+    - name: run Rally script
+      shell: ./run_rally.sh {{ build_tag }}
+      become: true
+      become_user: root
```

Comment:
I think we can do this more easily with the `script` module, which allows you to reference a local file to execute remotely. You can skip the manual transfer part.